routes/project/location/postgres.rb

```diff
@@ -70,7 +72,20 @@ class Clover
 
         pg.update(target_vm_size: target_vm_size.vm_size, target_storage_size_gib:, ha_type:)
 
-        Serializers::Postgres.serialize(pg, {detailed: true})
+        if api?
+          Serializers::Postgres.serialize(pg, {detailed: true})
+        else
+          flash["notice"] = "'#{pg.name}' will be updated according to requested configuration"
+          response["Location"] = "#{@project.path}#{pg.path}"
+          204
+        end
+      rescue CloverError => e
+        raise e if api?
+
+        flash["error"] = e.message
+        flash["errors"] = (flash["errors"] || {}).merge(e.details).transform_keys(&:to_s)
```

Comment:
We already perform similar rescue:
https://github.com/ubicloud/ubicloud/blob/28dacb968b9222e72bdc829c240ad3d272fb80c8/clover.rb#L226

But AJAX requests has early return

https://github.com/ubicloud/ubicloud/blob/28dacb968b9222e72bdc829c240ad3d272fb80c8/clover.rb#L214

I prefer not to duplicate rescues here. We can allow AJAX requests with a special header or let all PATCH requests continue instead of returning.